zero/src/rpc/jerigon.rs

```diff
@@ -51,7 +57,7 @@ where
         block_trace: BlockTrace {
             trie_pre_images: BlockTraceTriePreImages::Combined(CombinedPreImages {
                 compact: hex::decode(block_witness.strip_prefix("0x").unwrap_or(&block_witness))
-                    .context("invalid hex returned from call to eth_getWitness")?,
+                    .context("invalid hex returned from call to {WITNESS_ENDPOINT}")?,
```

Comment:
Note this will NOT do interpolation

Reply:
Right, @Nashtare you miss `:`

Reply:
You need to use the `format!` macro, you won't need the `:`. Right now this is a plain string

Reply:
Oops mb